src/internal/templates/internal/lore_wiki.html

```diff
@@ -0,0 +1,72 @@
+{% extends 'internal/base.html' %}
+{% load i18n %}
+{% load static %}
+
+
+{% block title %}
+    {% translate "The wiki of MAKE lore" %}
+{% endblock title %}
+
+{% block header_links %}
+    <link rel="stylesheet" href="{% static 'internal/css/make_lore.css' %}">
+    <link rel="stylesheet" href="{% static 'web/css/header.css' %}"/>
+    <script defer src="{% static 'web/js/header.js' %}"></script>
+{% endblock header_links %}
+
+{% block body %}
+    <div id="lore-page">
```

Comment:
Use a `<div>` with the classes `ui text container` as the outermost element, like pretty much all of the other templates do 🙂

Reply:
This messes up the entire look of the page, and gives me too large margins.

Reply:
Larger margins than on the other pages using `ui text container`? If they're the same, then that is very much the point, as [lines should not be too wide (seems like around 70-80 characters is the convention) for accessibility reasons](https://www.w3.org/WAI/tutorials/page-structure/styling/#line-length) 🙂 _(Though kind of ironic that the linked page does not have a `max-width` set for its text 😅)_

So for readability and a consistent layout across all the pages of the website, I'd recommend starting with `ui text container` as a baseline, and using CSS properties like `position: absolute/fixed/sticky` to place elements outside of the margins, if that's wanted - like [the "Topics" sidebar you posted a screenshot of above](https://github.com/MAKENTNU/web/pull/675#issuecomment-1711822749).